packages/kbn-expandable-flyout/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-browser",
   "id": "@kbn/expandable-flyout",
-  "owner": "@elastic/security-threat-hunting-investigations"
-}
+  "owner": [
+    "@elastic/security-threat-hunting-investigations"
+  ],
+  "group": "security",
+  "visibility": "private"
```

Comment:
this package was built with the intent of being usable by anyone, not just `security`... There is nothing in the code that ties it to any specific solution. Would it make sense to make it 
```
"group": "platform",
"visibility": "shared"
```
?

Reply:
Even if it's usable by anyone, we want to keep "platform" surface area as thin as possible (see [thread](https://elastic.slack.com/archives/C07TCKTA22E/p1731601159370569?thread_ts=1731581785.919069&cid=C07TCKTA22E)).
Thus, we prefer to categorise it as "security" as it's not used by other solutions ATM.

Reply:
Ok thanks for the thread link. I understand though don't fully agree with the decision. How would we let other solutions know that the package is fully reusable/completely solution agnostic if we mark it as `solution x - private`?

Anyway this is not a problem for this PR, approving now!